### PR TITLE
added stylling for products

### PR DESCRIPTION
**Pull request by developertom01**
I added styling for product section

### CODE REVIEW

File: source/_layouts/master.blade.php [hunk 1]
```diff
@@ -12,11 +12,10 @@
         <meta property="og:description" content="{{ $page->siteDescription }}" />
 
         <title>{{ $page->title ?  $page->title . ' | ' : '' }}{{ $page->siteName }}</title>
-
         <link rel="home" href="{{ $page->baseUrl }}">
         <link rel="icon" href="/favicon.ico">
         <link href="/blog/feed.atom" type="application/atom+xml" rel="alternate" title="{{ $page->siteName }} Atom Feed">
-
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
```
Comment (juliuskrah) [inline source/_layouts/master.blade.php:18]: You added font awesome to the npm package.

Do we need it as a cdn link also?

File: package.json [hunk 2]
```diff
@@ -20,5 +20,8 @@
         "tailwindcss": "^1.0.4",
         "vue": "^2.5.21",
         "vue-template-compiler": "^2.5.21"
+    },
+    "dependencies": {
+        "@fortawesome/fontawesome-free": "^5.14.0"
```
Comment (juliuskrah) [inline package.json:25]: Font awesome is added here. It is included the layout page as a cdn link.

We need to pick one approach and stick with it

File: source/contact.blade.php [hunk 3]
```diff
@@ -8,66 +8,119 @@
...
+            <h4>
+                You can reach us on ...
+            </h4>
+            </div>
+            <div>
+                <span class="icon">        
+                    <i class="fa fa-phone-square" aria-hidden="true"></i>
+                </span>
+                <span>  phone number:  </span>
+                <span>   +233-26-123-456  </span>
```
Comment (juliuskrah) [inline source/contact.blade.php:113]: Let's include the number from the configure file. This way the update is done once and it reflects on all pages

`{{ $page->contactNumber }}`

Look at how I did it on the staging branch of this repository.

I think you worked on the master branch

File: source/contact.blade.php [hunk 4]
```diff
@@ -8,66 +8,119 @@
...
+            </div>
+            <div>
+            <span class="icon">        
+                <i class="fa fa-phone-square" aria-hidden="true"></i>
+            </span>
+            <span>  email:  </span>
+            <span>    info@jaesoft.tech  </span>
```
Comment (juliuskrah) [inline source/contact.blade.php:120]: Use the email in the `Config.php`

`{{ $page->contactEmail }}`

**Review by juliuskrah — CHANGES_REQUESTED** [review]

File: source/_layouts/master.blade.php (hunk 1 above)
Comment (developertom01) [reply source/_layouts/master.blade.php:18]: I installed with npm but it was not working that is why I used the cdn. I will take the dependency off

File: source/contact.blade.php (hunk 3 above)
Comment (developertom01) [reply source/contact.blade.php:113]: okay

**Pull request closed without merging**